stack-easybib/libraries/helpers.rb

```diff
@@ -0,0 +1,21 @@
+module StackEasybib
+  class Helpers
```

Comment:
Maybe rename this to NginxHelpers or something along those lines, so we dont risk ending up a one-helper-for-everything library down the road ;)